react-app/src/redux/Auth/actions.js

```diff
@@ -0,0 +1,47 @@
+import { authService } from '../../services/authService';
+
+export const actionNames = {
+  GET_USER: 'GET_USER',
+  GET_USER_SUCCESS: 'GET_USER_SUCCESS',
+  GET_USER_FAILURE: 'GET_USER_FAILURE'
+};
+
+const responseBad = (message, dispatch) => {
```

Comment:
We usually define these actions in a private actions object, and call them after the action (for example, getUserFailure). Now you got just one action, but in a project you can have more than 50 actions in a single reducer, and action naming becomes an important part.